learn/getting-started/working-with-data/using-query-expressions-to-manipulate-data.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: using-query-expressions-to-manipulate-data
+title: Using Query Expressions to Manipulate Data
+permalink: /learn/using-query-expressions-to-manipulate-data
+description: This simple guide helps you understand how query expressions can be used on data to produce new data 
+active: using-query-expressions-to-manipulate-data
+intro: This simple guide helps you understand how query expressions can be used on data to produce new data 
+redirect_from:
+  - /learn/using-query-expressions-to-manipulate-data
+---
+
+This simple guide helps you understand how query expressions (a.k.a language integrated queries) can be used on data to produce new data.
+
+Ballerina supports first-class support for writing SQL-like queries to process data.
+
+Language-integrated queries can process any Ballerina iterable collection.
+
+In this tutorial, you will be writing queries to filter, sort, and join with other data sets and produce new data sets.
+
+1. Creating a new Ballerina project.
+2. Defining the Covid dataset to be processed.
+3. Finding all countries which have more than 10,000,000 Covid cases.
```

Comment:
Shall we include the operation itself in the heading.
eg:
`Filtering all countries which have more than 10,000,000 Covid cases.`
`Sorting the top three countries by the number of reported Covid cases.`

Reply:
Sure